nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RHostedRecipe.groovy

```diff
@@ -112,11 +110,12 @@ class RHostedRecipe
         .handler(hostedHandlers.putArchive)
         .create())
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
-        .handler(browseUnsupportedHandler)
+    builder.route(nonRFileUploadMatcher()
```

Comment:
add the security handler

Reply:
do we need exception handler?

Reply:
Added security handler in 47bd5fb9dc9524a372cb0a1cd8218dad8aa45f03